packages/sdk-codegen/src/sdkModels.ts

```diff
@@ -82,29 +90,29 @@ export interface ISymbol {
   search(rx: RegExp, criteria: SearchCriteria): boolean
 }
 
-export type IKeyedCollection<T> = Record<string, T>
-export type IMethodList = IKeyedCollection<IMethod>
-export type ITypeList = IKeyedCollection<IType>
-export type ITagList = IKeyedCollection<IMethodList>
-export type IPropertyList = IKeyedCollection<IProperty>
-export type IKeyList = Set<string>
+export type KeyedCollection<T> = Record<string, T>
```

Comment:
is this just a style/convention change or is something more significant happening here?

Reply:
Naming convention. Those originally were interfaces, then I changed them to types.